source/plugin.js

```diff
@@ -0,0 +1,39 @@
+const { autoTasks } = require('./internals/tasks');
+const { rlInterface } = require('./internals/options');
+const { executeTask } = require('./internals/executor');
+
+module.exports = [
+  {
+    description: 'fully automated project state clean: like a freshly-cloned, never-started repo',
+    name: 'clean-project-auto',
+    func: () => {
+
+      console.log('');
+      console.log('Executing fully-automated project clean.');
+      console.log('Use \'react-native clean-project\' for more control');
+      console.log('Use \`./node_modules/.bin/react-native-clean-project\' for total control');
+      console.log('');
+
+      rlInterface.close(); // if we don't do this it hangs waiting for input
+
+      let taskPromises = [];
+      autoTasks.forEach(task => {
+        taskPromises.push(executeTask(task));
+      });
+
+      Promise.all(taskPromises).then(() => {
```

Comment:
```suggestion
      return Promise.all(taskPromises).then(() => {
```

By returning the result from the `func` function CLI is able to intercept the error messages and know when the command succeeded/failed